pdr_backend/pdr_dashboard/dash_components/util.py

```diff
@@ -260,6 +267,47 @@ def get_predictoors_data_from_payouts(
     return user_payout_stats
 
 
+@enforce_types
+def get_feeds_stats_from_db(lake_dir: str):
+    feeds = _query_db(
+        lake_dir,
+        f"""
+            SELECT COUNT(DISTINCT(contract, pair, timeframe, source))
+            FROM {Prediction.get_lake_table_name()}
+        """,
+        scalar=True,
+    )
+
+    feeds = feeds[0] if feeds else 0
+
+    accuracy, revenue, volume = _query_db(
+        lake_dir,
+        f"""
+            SELECT
+                SUM(CASE WHEN payout > 0 THEN 1 ELSE 0 END) * 100.0 / COUNT(*) AS avg_accuracy,
+                SUM(payout) as total_revenue,
```

Comment:
The revenue should be calculated from subscription sales, how much revenue the feeds has brought from subscription sells